joint_limit_lower: 1.0
joint_limit_upper: -1.0